docs/installation/instructions.rst

```diff
@@ -1,10 +1,18 @@
 Installation Instructions
 =========================
 
+Supported Platforms
+-------------------
+
+Pulp has been tested to work on Ubuntu, Fedora, and CentOS. It should work on other operating systems
+such as Debian or OSX, but no guarentees.
+
 .. note::
 
     As Pulp 3 currently does not have an SELinux Policy, it currently requires the target
```

Comment:
What about mentioning AppArmor here too? It's the SELinux for Ubuntu.

Reply:
I know very little about AppArmor but I think it works differently?  I'm not sure there is anything to disable by default.